### ISSUE TITLE
add 4th field for location

### ISSUE THREAD

**Issue by tineml** [IFRCGo/cbs#1041]
need a 4th level/field for location with title "other". Not mandatory field. See uxpin:

https://preview.uxpin.com/6fe1059e581ead11550dd9bc9312290631e4f192#/pages/101789199/simulate/sitemap

**Comment by hodoea** [comment]
So @tineml the rest of the fields are required then except for "other"? Then I should reflect that in the design... Ok?

**Comment by tineml** [comment]
yes, @hodoea, that's correct and yes, great to have that clearly in the UI:)

**Comment by hodoea** [comment]
Done: https://preview.uxpin.com/6fe1059e581ead11550dd9bc9312290631e4f192#/pages/101789199/simulate/sitemap
